docs/src/main/paradox/running-the-application.md

```diff
@@ -0,0 +1,81 @@
+[info] Resolving jline#jline;2.14.3 ...
+[info] Done updating.
+[info] Compiling 1 Scala source and 1 Java source to /Users/x/hello-akka-scala/target/scala-2.12/classes...
+[success] Total time: 5 s, completed May 9, 2017 9:52:34 AM
+```
+
+### Running the Project
+
+Now it is time to run the project. We will run it using `sbt` by issuing the following command in a Terminal window:
+
+```
+sbt run
+```
+
+The output should look _something_ like this (scroll all the way to the right to see the Actor output):
+
+```
+[info] Loading global plugins from /Users/x/.sbt/0.13/plugins
+[info] Loading project definition from /Users/x/hello-akka-scala/project
+[info] Set current project to hello-akka-scala (in build file:/Users/x/hello-akka-scala)
+[info] Running com.lightbend.akka.sample.HelloAkka
+>>> Press ENTER to exit <<<
+[INFO] [05/09/2017 09:57:15.979] [helloakka-akka.actor.default-dispatcher-2] [akka://helloakka/user/printerActor] Greeting received (from Actor[akka://helloakka/user/howdyGreeter#-1854995773]): Howdy, Akka
+[INFO] [05/09/2017 09:57:15.980] [helloakka-akka.actor.default-dispatcher-2] [akka://helloakka/user/printerActor] Greeting received (from Actor[akka://helloakka/user/helloGreeter#-1072877049]): Hello, Scala
+[INFO] [05/09/2017 09:57:15.980] [helloakka-akka.actor.default-dispatcher-2] [akka://helloakka/user/printerActor] Greeting received (from Actor[akka://helloakka/user/goodDayGreeter#1972065097]): Good day, Play
+[INFO] [05/09/2017 09:57:15.980] [helloakka-akka.actor.default-dispatcher-2] [akka://helloakka/user/printerActor] Greeting received (from Actor[akka://helloakka/user/howdyGreeter#-1854995773]): Howdy, Lightbend
+```
+
+Remember that we set our `Printer` Actor to use the Logger from Akka? This is why there is a lot of extra information when we log things. The log output contains information like when and from what actor it was logged. Let's focus on the output from the `Printer` Actor for a while:
+
+```
+... Howdy, Akka
+... Hello, Scala
+... Good day, Play
+... Howdy, Lightbend
+```
+
+This is the result of our code that sends messages to the `Greeter` Actor:
+
+@@snip [HelloAkka.scala]($g8src$/scala/com/lightbend/akka/sample/HelloAkka.scala) { #main-send-messages }
+
+Try running the code a couple of more times and make sure to notice the order of the logging. Does it remain the same or does it change? As you can see it changes sometimes when you run. What's happening here? For the sake of brevity, we will not explain this in detail. For now, it is enough to say that when you send messages to Actors this happens asynchronously and as such we cannot rely on that things will happen in a certain order. Instead, we have to design our program with this in mind! This is a new mindset but rest assured that it is not rocket science and once you get it everything will become clear; just like for <a href="https://en.wikipedia.org/wiki/Neo_(The_Matrix)">Neo in the Matrix</a>.
```

Comment:
"we cannot rely on that things will happen in a certain order" is too scary, since messages for a specific actor is processed in order. Perhaps it's enough to say "... when you send messages to different Actors ..."